src/main/scala/io/radicalbit/nsdb/connector/kafka/sink/conf/Constants.scala

```diff
@@ -31,4 +31,11 @@ object Constants {
     final lazy val NamespaceFieldName = "namespace"
   }
 
+  object SemanticDelivery {
```

Comment:
is it possible to use an ADT to describe the possible semantic deliveries?

Reply:
I agree!